src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/IslHelper.groovy

```diff
@@ -0,0 +1,90 @@
+package org.openkilda.functionaltests.helpers
+
+import groovy.util.logging.Slf4j
+import org.openkilda.messaging.info.event.IslChangeType
+import org.openkilda.testing.model.topology.TopologyDefinition
+import org.openkilda.testing.model.topology.TopologyDefinition.Isl
+import org.openkilda.testing.service.northbound.NorthboundService
+import org.openkilda.testing.service.northbound.NorthboundServiceV2
+import org.openkilda.testing.tools.IslUtils
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.context.annotation.Scope
+import org.springframework.stereotype.Component
+
+import static groovyx.gpars.GParsExecutorsPool.withPool
+import static org.openkilda.messaging.info.event.IslChangeType.DISCOVERED
+import static org.openkilda.messaging.info.event.IslChangeType.FAILED
+import static org.springframework.beans.factory.config.ConfigurableBeanFactory.SCOPE_PROTOTYPE
+
+/**
+ * Holds utility methods for manipulating y-flows.
+ */
+@Component
+@Slf4j
+@Scope(SCOPE_PROTOTYPE)
+class IslHelper {
+    @Autowired
+    TopologyDefinition topology
+    @Autowired @Qualifier("islandNbV2")
+    NorthboundServiceV2 northboundV2
+    @Autowired @Qualifier("islandNb")
+    NorthboundService northbound
```

Comment:
Do we need topology and northbounds vars? It seems like we don't use them.

Reply:
Fair enough.